python/sglang/srt/entrypoints/openai/serving_chat.py

```diff
@@ -122,6 +123,23 @@ def _convert_to_internal_request(
             else:
                 prompt_kwargs = {"input_ids": processed_messages.prompt_ids}
 
+        customer_labels = None
+        if self.tokenizer_manager.server_args.tokenizer_metrics_custom_labels:
+            raw_labels = (
+                json.loads(raw_request.headers.get("x-customer-labels"))
```

Comment:
"x-customer-labels" can be a configurable item. This is because in certain scenarios, gateway headers may need to comply with specific rules, which can vary across different scenarios.

Reply:
btw, json.loads may throw exception, should be hanlde